dev/README.md

```diff
@@ -0,0 +1,41 @@
+# This directory contains information and tools to help develop the Coq system
+
+
+## Debugging and profiling (`dev/`)
+##### More info: [`doc/debugging.md`](doc/debugging.md)
+
+| File | Description |
+| ---- | ----------- |
+| dev/ocamldebug-coq | to launch ocaml debugger (generated by the configure script) |
+| dev/db | to install pretty-printers from ocaml debugger |
+| dev/base_db | to install raw pretty-printers from ocaml debugger |
+| dev/include | to install pretty-printers from ocaml toplevel (use with the coq Drop command) |
+| dev/base_include | to install raw pretty-printers from ocaml toplevel |
+| dev/vm_printers.ml, top_printers.ml | ML pretty-printers for debugging |
+
+
+## Miscellaneous information about the code (`dev/doc`)
+##### Beginner's guide: [`dev/doc/README.md`](doc/README.md)
+
+| File | Description |
+| ---- | ----------- |
+| [`dev/doc/changes.md`](doc/changes.md) | (partial) per-version summary of the evolution of Coq ML source |
+| [`dev/doc/style.txt`](doc/style.txt) | a few style recommendations for writing Coq ML files |
+| [`dev/doc/debugging.md`](/doc/debugging.md) | help for debugging or profiling |
+| [`dev/doc/universes.txt`](/doc/universes.txt) |  help for debugging universes |
+| [`dev/doc/translate.txt`](/doc/translate.txt) | help for using coq translator |
```

Comment:
This file is probably outdated, and could be archived / removed.

Reply:
@Zimmi48 which file, translate? The file is still there, though? Do you want me to **remove** `dev/doc/translate.txt` from the repo, as well, as remove the reference?

Reply:
Yes. It's probably time to remove this file.